src/System.Net.Http.Rtc/ref/System.Net.Http.Rtc.cs

```diff
@@ -1,14 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
This assembly was only for uap and we'll just harvest it?

Reply:
Is this really uap-specific, or is it actually to do with WinRT and something that could be used from outside of uap as well?

Reply:
It is complicated to explain what this is. :)

But it can not be used anymore in UAP apps. It was something that only worked for Windows 8 Apps.  Those apps still used .NET Framework actually and not .NET Core.  "RTC" was about AOAC (always-on-always-connected) machines. And it requires special hooks in .NET Framework to activate the COM interface.

When UAP was created, we needed to be able to have Windows 8 apps still compile but this functionality was turned off.  But the reference assembly remains.

I think this assembly can just be harvested.  The code is basically a no-op.

Reply:
`System.Net.Http.Rtc` is dead Windows Store specific contract. It is fine to delete it.